ios/Codeforces Watcher/Application/core/extensions/View+Extension.swift

```diff
@@ -0,0 +1,8 @@
+import SwiftUI
+
+extension View {
```

Comment:
If `RoundedCorner` is used only here, you can move it to this extension and rename it to `View+RoundedCorners`.